packages/communication-react/CHANGELOG.beta.md

```diff
@@ -4,123 +4,71 @@ This log was last generated on Mon, 28 Aug 2023 22:17:59 GMT and should not be m
-- Fetch translated strings ([PR #3431](https://github.com/azure/communication-ui-library/pull/3431) by 98852890+vhuseinova-msft@users.noreply.github.com)
-- Updating to include latest Calling SDK Hotfix ([PR #3482](https://github.com/azure/communication-ui-library/pull/3482) by alkwa@microsoft.com)
-- Updated Beachball config with new prompts ([PR #3371](https://github.com/azure/communication-ui-library/pull/3371) by joshlai@microsoft.com)
-- Bump version to 1.7.0-beta.2 ([PR #3509](https://github.com/azure/communication-ui-library/pull/3509) by jiangnanhello@live.com)
-- Updating signaling sdk to 1.0.0-beta19 ([PR #3414](https://github.com/azure/communication-ui-library/pull/3414) by joshlai@microsoft.com)
+
 ### Bug Fixes
-- Add stream size to remote video stream ([PR #3516](https://github.com/azure/communication-ui-library/pull/3516) by 97124699+prabhjot-msft@users.noreply.github.com)
-- disable raise hand button in lobby ([PR #3512](https://github.com/azure/communication-ui-library/pull/3512) by ruslanzdor@microsoft.com)
-- Fix nightly alpha releases having || in the version dependencies ([PR #3440](https://github.com/azure/communication-ui-library/pull/3440) by 2684369+JamesBurnside@users.noreply.github.com)
-- Remove and revert storybook ([PR #3513](https://github.com/azure/communication-ui-library/pull/3513) by 9044372+JoshuaLai@users.noreply.github.com)
 - Fix image gallery double click to use default behaviour ([PR #3515](https://github.com/azure/communication-ui-library/pull/3515) by 9044372+JoshuaLai@users.noreply.github.com)
-- Adding new image gallery conditional conditional flage ([PR #3507](https://github.com/azure/communication-ui-library/pull/3507) by 9044372+JoshuaLai@users.noreply.github.com)
-- Improve shake to send logs: capture all console and window logging events, do not log on desktop, fix iOS asking for permissions everytime page is opened ([PR #3486](https://github.com/azure/communication-ui-library/pull/3486) by 2684369+JamesBurnside@users.noreply.github.com)
-- Remove redundant cc comment ([PR #3455](https://github.com/azure/communication-ui-library/pull/3455) by jiangnanhello@live.com)
-- Fix storybook bot missing displayName ([PR #3490](https://github.com/azure/communication-ui-library/pull/3490) by 77021369+jimchou-dev@users.noreply.github.com)
-- Fixed logic of when to show display name field in Calling sample app ([PR #3407](https://github.com/azure/communication-ui-library/pull/3407) by 79475487+mgamis-msft@users.noreply.github.com)
-- Fixes side pane to refocus the people button when user closes the people pane with keyboard. ([PR #3403](https://github.com/azure/communication-ui-library/pull/3403) by 94866715+dmceachernmsft@users.noreply.github.com)
+- Fix logic of when to show display name field in Calling sample app ([PR #3407](https://github.com/azure/communication-ui-library/pull/3407) by 79475487+mgamis-msft@users.noreply.github.com)
```

Comment:
I don't think we should have this here since this is a fix for the Calling sample which we don't package for npm